parser.ts

```diff
@@ -188,15 +196,70 @@ export function traverseExpr(c: TreeCursor, s: string): Expr<null> {
     case "MemberExpression":
       c.firstChild(); // Focus on object
       var objExpr = traverseExpr(c, s);
-      c.nextSibling(); // Focus on .
-      c.nextSibling(); // Focus on property
-      var propName = s.substring(c.from, c.to);
-      c.parent();
-      return {
-        tag: "lookup",
-        obj: objExpr,
-        field: propName,
-      };
+      c.nextSibling(); // Focus on . or [
```

Comment:
@ricky8554 @erauyeung can you compare this to your bracket lookup solution and see what a good harmonized solution is?